packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -0,0 +1,107 @@
+    filterPattern: string
+}
+
+export async function tailLogGroup(logData?: { regionName: string; groupName: string }): Promise<void> {
+    const wizard = new TailLogGroupWizard(logData)
+    const wizardResponse = await wizard.run()
+    if (!wizardResponse) {
+        throw new CancellationError('user')
+    }
+
+    //TODO: Remove Log. For testing while we aren't yet consuming the wizardResponse.
+    getLogger().info(JSON.stringify(wizardResponse))
+}
+
+export class TailLogGroupWizard extends Wizard<TailLogGroupWizardResponse> {
+    public constructor(logGroupInfo?: CloudWatchLogsGroupInfo) {
+        super({
+            initState: {
+                regionLogGroupSubmenuResponse: logGroupInfo
+                    ? {
+                          data: logGroupInfo.groupName,
+                          region: logGroupInfo.regionName,
+                      }
+                    : undefined,
+            },
+        })
+        this.form.regionLogGroupSubmenuResponse.bindPrompter(createRegionLogGroupSubmenu)
+        this.form.logStreamFilter.bindPrompter((state) => {
+            if (!state.regionLogGroupSubmenuResponse?.data) {
+                throw Error('LogGroupName is null')
+            }
+            return new LogStreamFilterSubmenu(
+                state.regionLogGroupSubmenuResponse.data,
+                state.regionLogGroupSubmenuResponse.region
+            )
+        })
+        this.form.filterPattern.bindPrompter((state) => createFilterPatternPrompter())
+    }
+}
+
+export function createRegionLogGroupSubmenu(): RegionSubmenu<string> {
+    return new RegionSubmenu(
+        getLogGroupQuickPickOptions,
+        {
+            title: localize('AWS.cwl.tailLogGroup.logGroupPromptTitle', 'Select Log Group to tail'),
+            buttons: [createExitButton()],
+        },
+        { title: localize('AWS.cwl.tailLogGroup.regionPromptTitle', 'Select Region for Log Group') },
+        'LogGroups'
+    )
+}
+
+async function getLogGroupQuickPickOptions(regionCode: string): Promise<DataQuickPickItem<string>[]> {
+    const client = new DefaultCloudWatchLogsClient(regionCode)
+    const logGroups = client.describeLogGroups()
+
+    const logGroupsOptions: DataQuickPickItem<string>[] = []
+
+    for await (const logGroupObject of logGroups) {
+        if (!logGroupObject.arn || !logGroupObject.logGroupName) {
+            throw Error('LogGroupObject name or arn undefined')
+        }
+
+        logGroupsOptions.push({
+            label: logGroupObject.logGroupName,
+            data: formatLogGroupArn(logGroupObject.arn),
+        })
+    }
+
+    return logGroupsOptions
+}
+
+function formatLogGroupArn(logGroupArn: string): string {
+    return logGroupArn.endsWith(':*') ? logGroupArn.substring(0, logGroupArn.length - 2) : logGroupArn
+}
+
+export function createFilterPatternPrompter() {
+    const helpUri = 'https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html'
```

Comment:
I think we've typically put documentation urls in `packages/core/src/shared/constants.ts`